spec/functions/extlib/ip_to_reverse_spec.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe 'extlib::ip_to_reverse' do
+  it 'exists' do
+    is_expected.not_to be_nil
+  end
+
+  describe 'IPv4' do
+    context 'with an IPv4 address' do
+      it { is_expected.to run.with_params('192.0.2.1').and_return('1.2.0.192.in-addr.arpa') }
+    end
+
+    context 'with an IPv4 address with CIDR' do
+      it { is_expected.to run.with_params('192.0.2.0/24').and_return('0.2.0.192.in-addr.arpa') }
+    end
+  end
+
+  describe 'IPv6' do
+    context 'with an IPv6 address' do
+      it { is_expected.to run.with_params('1080:0:0:0:8:800:200C:417A').and_return('a.7.1.4.c.0.0.2.0.0.8.0.8.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.8.0.1.ip6.arpa') }
+    end
+
+    context 'with an IPv6 address with CIDR' do
+      it { is_expected.to run.with_params('1081:0::0/64').and_return('0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.1.8.0.1.ip6.arpa') }
+    end
```

Comment:
It's good that this is using the IPv4 documentation prefix, should also use the IPv6 documentation prefix: https://en.wikipedia.org/wiki/Reserved_IP_addresses

Reply:
Done